erratatool.yml

```diff
@@ -65,7 +65,7 @@ solution: |
   (For aarch64 architecture)
   The image digest is sha256:<SHASUM_HERE>
 
-  All OpenShift Container Platform 4.14 users are advised to upgrade to these updated packages and images when they are available in the appropriate release channel. To check for available updates, use the OpenShift Console or the CLI oc command. Instructions for upgrading a cluster are available at  https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating-cluster-cli.html
+  All OpenShift Container Platform 4.14 users are advised to upgrade to these updated packages and images when they are available in the appropriate release channel. To check for available updates, use the OpenShift Console or the CLI oc command. Instructions for upgrading a cluster are available at  https://docs.openshift.com/container-platform/4.14/updating/updating-cluster-cli.html
```

Comment:
This was update here: https://github.com/openshift-eng/ocp-build-data/pull/3469

Reply:
Ah so we don't need this PR?